src/main/app/src/Components/Landscape/Map/Map.tsx

```diff
@@ -36,79 +51,59 @@ interface SVGData {
 }
 
 /**
- * Displays a choosen landscape as interactive SVG
- * @param identifier Landscape Identifier
- * @param onItemClick Handler for our label click
+ * Displays a chosen landscape as interactive SVG
+ *
+ * @param setSidebarContent function to set sidebar/drawer content
+ * @param setFindFunction function to use to find an item. make sure to pass an anon func returning the actually used function
+ * @param setPageTitle can be used to set the page title in parent state
  */
-const Map: React.FC<Props> = () => {
+const Map: React.FC<Props> = ({ setSidebarContent, setFindFunction, setPageTitle }) => {
```

Comment:
The sidebar is more like a card. I would call it `InformationCard`or something else than "sidebar". I think sidebar is actually a bar at the right or left side of the page.

Reply:
As mentioned the sidebar indeed is a container for content, mostly cards, and goes from top to bottom 😄